charts/newrelic-infrastructure/values.yaml

```diff
@@ -36,7 +36,7 @@ images:
   # @default -- See `values.yaml`
   integration:
```

Comment:
could you bump the infrastructure bundle and forwareder images?

Reply:
Done in c186747d4b0dcb487aa85c04755df4e2b3c32397.